subsys/net/lib/lwm2m/Kconfig

```diff
@@ -87,6 +87,11 @@ config LWM2M_ENGINE_MAX_OBSERVER
 	  This value sets the maximum number of resources which can be
 	  added to the observe notification list.
 
+config LWM2M_ENGINE_NOTIFICATION_TIMEOUT_REGISTRATION_UPDATE
```

Comment:
Thanks for the PR! Could you please shorten this option name? This is way too long

Reply:
LWM2M_ENGINE_TIMEOUT_REG_UPDATE okay?